docs/examples/3rdparty_analysis_workflow.sh

```diff
@@ -77,7 +77,7 @@ bash code/run_analysis.sh ||true
 
 datalad unlock
 bash code/run_analysis.sh
-datalad save -a "Alice always helps"
+datalad save -a -m "Alice always helps"
```

Comment:
Will come later when actual narrative arrives. I was planing to do this prior last publish.